TipCalculator/MainViewController.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  ViewController.swift
+//  TipCalculator
+//
+//  Created by vmalikov on 1/30/16.
+//  Copyright © 2016 justForFun. All rights reserved.
+//
+
+import UIKit
+
+extension Float {
+    func format(f: String) -> String {
+        return NSString(format: "$%\(f)f", self) as String
+    }
+}
+
+class MainViewController: UIViewController, UITextFieldDelegate {
```

Comment:
Since view controllers become larger, it will make sense to separate them into meaningful parts. I usually separate delegate methods to be an extension of the class (even though I put them all in the same file, it is easier to read the code). 
So I would separate all code related to `UITextFieldDelegate` to be `extension MainViewController: UITextFieldDelegate {...}` and would put it under this class in the same file. 
What do you think?